frontend/src/features/dashboard/dashboardSlice.ts

```diff
@@ -15,7 +15,7 @@ const slice = createSlice({
     // Main
     setCurrentTab: (state, action) => {
       const id = action.payload
-      if (state.main.tabs.hasOwnProperty(id)) {
+      if (id in state.main.tabs) {
```

Comment:
As far as I remember these two are not completely equivalent. The `in` operator is less restrictive as it allows for matches even in the parents classes (see [here](https://stackoverflow.com/a/13633027)).